src/figvalidationservice/figcrpords.yaml

```diff
@@ -327,5 +334,16 @@ definitions:
         type: string
       status_message:
         type: string
-
- 
\ No newline at end of file
+  validateOrgApplicantServiceOrdsResponse:
+    type: object
+    required:
+    - validation_result
+    - status_code
+    - status_message
+    properties:
+      validation_result:
+        type: string
+      status_code:
+        type: string
+      status_message:
+        type: string   
```

Comment:
Needs newline at bottom. Thanks.

Reply:
Renamed response class name and also added new line

Reply:
Thank you